manifests/p/Puppet/pdk/2.7.0/Puppet.pdk.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-19041-1682
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Puppet.pdk
+PackageVersion: 2.7.0
+PackageLocale: en-US
+Publisher: Puppet Inc
+PublisherUrl: https://www.puppet.com/ 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+Author: Puppet Inc 
+PackageName: Puppet Development Kit
+PackageUrl: https://github.com/puppetlabs/pdk
+License: Apache 2.0
+LicenseUrl: https://github.com/puppetlabs/pdk/blob/main/LICENSE
+# Copyright: 
+# CopyrightUrl: 
+ShortDescription: Puppet.pdk
```

```suggestion
ShortDescription: |
  The Puppet Development Kit (PDK) includes key Puppet code development and testing tools for
  Linux, Windows, and OS X workstations, so you can install one package with the tools
  you need to create and validate new modules.
```
Need a better ShortDescription